gateway/src/context.rs

```diff
@@ -37,7 +39,21 @@ impl ServerContext {
             OnceLock::new()
         };
 
+        const START_LATENCY_DECADE: i16 = -6;
+        const END_LATENCY_DECADE: i16 = 3;
```

Comment:
Oh, maybe uploading an update image does take a long time! This might actually be _low_ then! In general, I'd probably pick about an order of magnitude higher than you could reasonably imagine a request taking, or high enough that lumping everything together is "fine" and not something you care to distinguish.

Reply:
To be entirely honest, I just snagged them from Nexus _because_ I don't have a super clear sense for what kinds of latency to expect in MGS. Perhaps the API endpoints for updating SP firmware can take multiple seconds? @jgallagher what do you think?

Reply:
Updates are broken up into "start" / "status" / "abort", each of which should be _relatively_ quick. (Callers are expected to poll "status" until the update is complete, in general.)

However, _resetting_ an SP can take a while, especially if it's a sidecar, because we wait for it to come back online before returning. The timeout for this is currently [30 seconds](https://github.com/oxidecomputer/management-gateway-service/blob/b273fec31891ab033957058701d2f499148e3f27/gateway-sp-comms/src/single_sp.rs#L107-L114) _per attempt_, with some small number of retries possible. In practice I think a sidecar reset is currently around 15 seconds, assuming success.

Reply:
Mmm, so I suppose that if we retry a reset three times before deciding it's failed, that's about 90 seconds...so I think let's leave this the way it is!